uint16 motor_left
uint16 motor_right
uint8 rake_flags
uint8 status